packages/mobile/src/identity/verification.ts

```diff
@@ -319,17 +330,24 @@ export function* doVerificationFlow(withoutRevealing: boolean = false) {
       }
 
       yield put(setVerificationStatus(VerificationStatus.CompletingAttestations))
+      // NOTE: I think we want the DEK registration out of the race because
+      // we want that to happen no matter what
       yield race({
-        actionableAttestationCompleted: all([
-          call(completeAttestations, attestationsWrapper, account, phoneHashDetails, attestations),
-          // Set acccount and data encryption key in Accounts contract
-          // This is done in other places too, intentionally keeping it for more coverage
-          call(registerAccountDek, account),
-        ]),
+        actionableAttestationCompleted: call(
+          completeAttestations,
+          attestationsWrapper,
+          account,
+          phoneHashDetails,
+          attestations
+        ),
         // This is needed, because we can have more actionableAttestations than NUM_ATTESTATIONS_REQUIRED
         requiredAttestationsCompleted: call(requiredAttestationsCompleted),
       })
 
+      // Set acccount and data encryption key in Accounts contract
+      // This is done in other places too, intentionally keeping it for more coverage
+      yield call(registerAccountDek, account)
```

Comment:
Suggest making this use `spawn` because we don't want to wait for it (we are not dependent on the results) and we will not raise an error if it fails.
```suggestion
      yield spawn(registerAccountDek, account)
```